util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -98,3 +101,23 @@ def remove_url_credentials(url: str) -> str:
     except ValueError:  # an unparseable url was passed
         pass
     return url
+
+
+def normalise_request_header_name(header):
```

Comment:
```suggestion
def normalise_request_header_name(header: str) -> str:
```

Reply:
Novice question, when is it recommended to use type hints?

Reply:
I would recommend we do this for all public apis. There are many reasons but I personally find them useful as a  documentation.